### ISSUE TITLE
getState | this.device.state | [object Object]

### ISSUE THREAD

**Issue by nicoh88** [nicoh88/homebridge-xiaomi-roborock-vacuum#91]
```
[2020-1-28 14:45:01] [Staubsauger EG Flur] DEB ensureDevice | roborock.vacuum.s5 | The socket is still on. Reusing it.
[2020-1-28 14:45:01] [Staubsauger EG Flur] DEB getState | roborock.vacuum.s5 | State {"batteryLevel":100,"charging":true,"cleaning":false,"fanSpeed":100}
[2020-1-28 14:45:01] [Staubsauger EG Flur] MON changedCleaning | roborock.vacuum.s5 | CleaningState is now false
[2020-1-28 14:45:01] [Staubsauger EG Flur] INF changedCleaning | roborock.vacuum.s5 | Cleaning is OFF.
[2020-1-28 14:45:01] [Staubsauger EG Flur] MON changedCharging | roborock.vacuum.s5 | ChargingState is now true
[2020-1-28 14:45:01] [Staubsauger EG Flur] INF changedCharging | roborock.vacuum.s5 | Charging is active
[2020-1-28 14:45:01] [Staubsauger EG Flur] INF changedCharging | roborock.vacuum.s5 | Robot was docked.
[2020-1-28 14:45:01] [Staubsauger EG Flur] MON changedSpeed | roborock.vacuum.s5 | FanSpeed is now 100%
[2020-1-28 14:45:01] [Staubsauger EG Flur] INF changedSpeed | roborock.vacuum.s5 | Speed was changed to 100% (Max), for HomeKit 100%
[2020-1-28 14:45:01] [Staubsauger EG Flur] DEB changedBattery | roborock.vacuum.s5 | BatteryLevel 100%
[2020-1-28 14:45:01] [Staubsauger OG Bad] DEB ensureDevice | rockrobo.vacuum.v1 | The socket is still on. Reusing it.
[2020-1-28 14:45:01] [Staubsauger OG Bad] DEB changedError | rockrobo.vacuum.v1 | ErrorID: charger-offline, ErrorDescription: Charger is offline
[2020-1-28 14:45:01] [Staubsauger OG Bad] WAR changedError | rockrobo.vacuum.v1 | Robot has an ERROR - charger-offline, Charger is offline
[2020-1-28 14:45:01] [Staubsauger OG Bad] ERR getState | this.device.state | [object Object]
```

ErrorID, but actually there is no error. 
`getState` output is not properly.

**Comment by afharo** [comment]
Thank you for reporting it! I'll have a look at it!

**Comment by nicoh88** [comment]
My son has been crawling recently and is always working on the robot. :-D

When "`charger-offline`", again, all devices in the homebridge instance no longer respond. I can reproduce that with version 0.6.5, 0.6.4, 0.6.3, 0.6.2, ...

**Comment by afharo** [comment]
Just to be clear. In order to reproduce it, I only need to unplug the charger from the wall socket, right?

**Comment by nicoh88** [comment]
I am currently not at home whether the charger is unplugged. But yes, I think so.

**Comment by nicoh88** [comment]
Fixed, waiting on version 0.6.6 or test the #master (0.6.6-dev) branch.

```
sudo su -
npm install -g nicoh88/homebridge-xiaomi-roborock-vacuum#master

```

**Comment by nicoh88** [comment]
THX @afharo 🚀
